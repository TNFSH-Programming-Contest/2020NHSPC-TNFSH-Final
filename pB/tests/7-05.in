27
-49.0 -840.5
-241.5 -806.6
-421.0 -729.2
-577.8 -612.4
-703.5 -462.7
-791.2 -288.0
-836.3 -97.7
-836.3 97.7
-791.2 288.0
-703.5 462.7
-577.8 612.4
-421.0 729.2
-241.5 806.6
-49.0 840.5
146.2 829.2
333.5 773.1
502.8 675.4
645.0 541.2
752.4 377.9
819.3 194.2
842.0 0.0
819.3 -194.2
752.4 -377.9
645.0 -541.2
502.8 -675.4
333.5 -773.1
146.2 -829.2
-377.2 137.3
