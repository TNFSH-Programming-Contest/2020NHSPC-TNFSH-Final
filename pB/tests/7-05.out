4842779.213
